Objects/abstract.c

```diff
@@ -1216,7 +1235,7 @@ PyNumber_InPlacePower(PyObject *v, PyObject *w, PyObject *z)
 {
     if (v->ob_type->tp_as_number &&
         v->ob_type->tp_as_number->nb_inplace_power != NULL) {
-        return ternary_op(v, w, z, NB_SLOT(nb_inplace_power), "**=");
+        return ternary_iop(v, w, z, NB_SLOT(nb_inplace_power), NB_SLOT(nb_power), "**=");
```

Comment:
`PyNumber_InPlacePower` can now just mirror the other `PyNumberInPlace*` functions, right?

So this is the only line that's needed here.

Reply:
Ah yes, thank you.